services/vidispine/chapter_test.go

```diff
@@ -82,9 +87,50 @@ func Test_GetChapterMetaForClips(t *testing.T) {
 	out, err := GetChapterMetaForClips(client, clips)
 	assert.NoError(t, err)
 	assert.NotEmpty(t, out)
+	assert.Len(t, out, 1)
 
 	for i, chapter := range out {
 		assert.GreaterOrEqual(t, len(chapter.Meta.Terse["title"]), 1, "Chapter in loop iteration %d has no title", i)
 	}
+}
+
+// This is an edge case where the annotations overlap by a few frames
+func Test_GetChapterMetaForClips_Overlapping2(t *testing.T) {
```

Comment:
clarify difference between the two tests?

Reply:
Merged the tests now